compiler/luci-interpreter/src/kernels/FullyConnected.cpp

```diff
@@ -36,27 +36,53 @@ FullyConnected::FullyConnected(const Tensor *input, const Tensor *weights, const
 
 void FullyConnected::configure()
 {
-  if (weights()->element_type() != DataType::FLOAT32)
-    throw std::runtime_error("Unsupported type.");
-
-  assert(input()->element_type() == DataType::FLOAT32);
-  assert(weights()->element_type() == DataType::FLOAT32);
-  assert(bias() == nullptr || bias()->element_type() == DataType::FLOAT32);
+  if (weights()->element_type() == DataType::U8 || weights()->element_type() == DataType::S8)
+  {
+    LUCI_INTERPRETER_CHECK(input()->element_type() == DataType::U8 ||
+                           input()->element_type() == DataType::S8);
+    LUCI_INTERPRETER_CHECK(output()->element_type() == DataType::U8 ||
+                           output()->element_type() == DataType::S8);
+    LUCI_INTERPRETER_CHECK(!bias() || bias()->element_type() == DataType::S32 ||
+                           bias()->element_type() == DataType::S64)
+  }
```

Comment:
These checks do not correspond to the implementation. `evalQuantized` only works with U8 input / output / weights and S32 bias.

Reply:
Okay, I will fix it!